HACKERS

```diff
@@ -14,8 +14,8 @@ lying around in this distribution.
 =head2 How to backport something
 
 First, make sure that what you want to backport is documented.  If it's worth
-backporting, it surely is worth documenting.  Send a documentation patch to 
-L<perlbug@perl.org|mailto:perlbug@perl.org> if necessary.  Also,
+backporting, it surely is worth documenting.  Submit a documentation patch to
+L<https://github.com/Perl/perl5/issues/new/choose> if necessary.  Also,
```

Comment:
probably just /issues uri I would not use the new/choose path

Reply:
ok